src/SDKs/Azure.Base/data-plane/Azure.Base/Http/HttpPipelineMethod.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Azure.Base.Http
 {
-    public enum HttpVerb : byte
+    public enum HttpPipelineMethod : byte
```

Comment:
Personally, If we're avoiding `HttpMethod` because of the `System.Net.Http` namespace, I'd prefer `HttpVerb` sticks around.   I find `HttpPipelineMethod` to be a bit less clear.

Reply:
Verb is not a commonly used term both in .net (it's `Method` in classic ASP.NET, ASP.NET Core, HttpClient and almost everywhere else) and in general (RFCs) so I argue that `HttpPipelineMethod ` is clearer especially considering `HttpPipeline*` is a common prefix

Reply:
I prefer `Method` to `Verb` but the first thing that caught in my mind was "why isn't this just an `HttpMethod`?  What makes it special to the pipeline?"

Reply:
> Could we consider disambiguating from System.Net.Http by just the namespace and call it HttpMethod?

I'm not sure about this, feels like there would be quite a few instances where both namespaces would be referenced in the same file, including our own code.

Reply:
Fair enough.   I agree that adapting to the new name would be less awkward than having to alias namespaces or disambiguate by typing them out as qualifiers.